common/src/test/java/bio/terra/catalog/iam/SamClientTest.java

```diff
@@ -11,7 +11,7 @@
 class SamClientTest {
   private static final String BASE_PATH = "basepath";
   private static final String TOKEN = "token";
-  private static final String AUTH_NAME = "b2coauth";
+  private static final String AUTH_NAME = "googleoauth";
```

Comment:
This change was required due to updating the sam client version. The test could instead use `getAuthentications()` and search for the token, if we wanted to be resilient to this kind of change.

Reply:
That would probably be cleaner, would you be willing to file that as a tech debt ticket?

Reply:
I got a fix working but it was a bit wordier than I liked, so I created a ticket: https://broadworkbench.atlassian.net/browse/DC-737